api/swagger.yaml

```diff
@@ -2790,6 +2790,24 @@ definitions:
           type: "array"
           items:
             $ref: "#/definitions/Mount"
+
+  Driver:
+    description: "Driver represents a driver (network, logging, secrets)."
+    type: "object"
+    required: [Name]
+    properties:
+      Name:
+        type: "string"
+        x-nullable: false
+        example: "some-driver"
+      Options:
```

Comment:
Consider 
`description: "key/value map of driver specific options."` (same as `DriverConfig.Options)`